addons/l10n_mx/models/res_config_settings.py

```diff
@@ -1,9 +1,18 @@
 # -*- coding: utf-8 -*-
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
     module_l10n_mx_edi = fields.Boolean('Mexican Electronic Invoicing')
+
+    @api.model
+    def get_views(self, views, options=None):
+        # HACK to not show the view since it uninstalls the module if unticked!
+        # 'module_l10n_mx_edi' will be removed in master.
+        mx_view = self.env.ref('l10n_mx_edi.res_config_settings_view_form', raise_if_not_found=False).sudo()
```

Comment:
After testing I'm still have the view, it shouldn't be this?
```suggestion
        mx_view = self.env.ref('l10n_mx.res_config_settings_view_form', raise_if_not_found=False).sudo()
```
In that case, I won't see the field